rosa_getting_started/rosa-quickstart.adoc

```diff
@@ -0,0 +1,171 @@
+
+
+//This content is pulled from rosa-getting-started-verify-elb-role.adoc
+[discrete]
+include::modules/rosa-getting-started-verify-elb-role.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-getting-started-verify-aws-quota.adoc
+[discrete]
+include::modules/rosa-getting-started-verify-aws-quota.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-sts-creating-a-cluster-quickly.adoc
+[id="rosa-quickstart-creating-a-cluster"]
+== Creating a ROSA cluster with AWS STS using the default auto mode
+
+The procedures in this document use the `auto` modes in the {cluster-manager} Hybrid Cloud Console to immediately create the required IAM resources using the current AWS account. The required resources include the account-wide IAM roles and policies, Operator policies, cluster-specific Operator roles, and OpenID Connect (OIDC) identity provider.
+
+//This content is pulled from rosa-sts-creating-a-cluster-quickly-ocm.adoc
+When using {cluster-manager-first} to create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can select the default options to create the cluster quickly.
+
+Before you can use {cluster-manager} to deploy ROSA with STS clusters, you must associate your AWS account with your Red Hat organization and create the required account-wide STS roles and policies.
+
+
+//This content is pulled from rosa-sts-overview-of-the-default-cluster-specifications.adoc
+[discrete]
+include::modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-sts-understanding-aws-account-association.adoc
+[discrete]
+include::modules/rosa-sts-understanding-aws-account-association.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-sts-associating-your-aws-account.adoc
+[discrete]
+include::modules/rosa-sts-associating-your-aws-account.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-sts-creating-account-wide-sts-roles-and-policies.adoc
+[discrete]
+include::modules/rosa-sts-creating-account-wide-sts-roles-and-policies.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-sts-creating-a-cluster-using-defaults-ocm.adoc
+[discrete]
+include::modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc[leveloffset=+2]
+
+////
+.Additional resources
+
+* For information about the account-wide IAM roles and policies that are required for ROSA deployments that use STS, see xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies_rosa-sts-about-iam-resources[Account-wide IAM role and policy reference].
+* For details about using the `auto` and `manual` modes to create the required STS resources, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-understanding-deployment-modes_rosa-sts-creating-a-cluster-with-customizations[Understanding the auto and manual deployment modes].
```

Comment:
Should "auto" and "manual" be single quotations or double? If I'm interpreting IBM styleguide correctly, it's double. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks

Reply:
`auto` and `manual` are in backticks, so they're good to go. Thanks!